scripts/link_directory_metadata.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+import basedosdados as bd
+import pandas as pd
+
+from pipelines.utils.metadata.utils import get_headers
+
+
+def get_columns(column_name: str, backend) -> pd.DataFrame:
```

Comment:
Você adicionou alguns tipo mas pode tipar o param `backend`
```suggestion
def get_columns(column_name: str, backend: bd.Backend) -> pd.DataFrame:
```